Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY version/ version/
 
 # Build
 ARG LD_FLAGS
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="$LD_FLAGS" -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -ldflags="$LD_FLAGS" -a -o manager main.go
```

Comment:
With `GOARCH` hardcoded to `amd64`, the build command creates binary for amd64 on my mac(arm64) local machine.  If we don't specify anything, it will default to the host arch on which it builds.

Reply:
We should update docker build commands in Makefile to use `--platform` flag to make cross-buidling of images easy.